src/commands/app/db/collection/create.js

```diff
@@ -0,0 +1,185 @@
+/*
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { DBBaseCommand } from '../../../../DBBaseCommand.js'
+import { Args, Flags } from '@oclif/core'
+import chalk from 'chalk'
+
+export class CreateCollection extends DBBaseCommand {
+  async run () {
+    const { collectionName } = this.args
+    const { collation, validator } = this.flags
+
+    try {
+      // Validate flags
+      this.validateFlags()
+
+      this.log(chalk.blue(`Creating collection '${collectionName}'...`))
+
+      // Log flag values if set
+      if (collation) {
+        this.log(chalk.dim(`   Using collation: ${collation}`))
+      }
+
+      if (validator) {
+        this.log(chalk.dim(`   Using validator: ${validator}`))
+      }
+
+      const client = await this.db.connect()
+
+      // Check if collection already exists
+      const existingCollections = await client.listCollections()
+      const collectionExists = existingCollections && existingCollections.some(col => col.name === collectionName)
+
+      if (collectionExists) {
+        const errorMessage = `Collection '${collectionName}' already exists`
+
+        this.log(chalk.red(errorMessage))
+        this.log(chalk.dim(`   Namespace: ${this.rtNamespace}`))
+
+        this.error(errorMessage)
+      }
+
+      // Build collection options
+      const options = {}
+      if (collation) {
+        options.collation = collation
+      }
+      if (validator) {
+        // Parse validator if it's a JSON string
+        if (typeof validator === 'string') {
+          try {
+            options.validator = JSON.parse(validator)
+          } catch (error) {
+            this.error(`Invalid validator JSON: ${error.message}`)
+          }
+        } else {
+          options.validator = validator
+        }
+      }
+
+      // Create the collection - only pass options if they exist
+      const result = Object.keys(options).length > 0
```

Comment:
This check is unnecessary, empty object is fine for options, in fact it's the default that lib-db uses anyways.